src/hierarchies/lin_reg_uni_hierarchy.cc

```diff
@@ -43,9 +43,14 @@ void LinRegUniHierarchy::save_posterior_hypers() {
 
 LinRegUniHierarchy::Hyperparams LinRegUniHierarchy::normal_invgamma_update()
     const {
+  if (card == 0) {  // no update possible
+    return *hypers;
+  }
+  // Compute posterior hyperparameters
   Hyperparams post_params;
-
   post_params.var_scaling = covar_sum_squares + hypers->var_scaling;
+  post_params.var_scaling_inv =
+      stan::math::inverse_spd(post_params.var_scaling);
   post_params.mean = post_params.var_scaling.llt().solve(
```

Comment:
since we're inverting `post_params.var_scaling`, we might actually use it to compute the mean.

If you seek the ultimate performance, we could also have

```
auto ldlt = post_params.var_scaling.ldlt()
post_params.var_scaling_inv = ldlt.solve(
      Eigen::MatrixXd::Identity(m.rows(), m.cols()));
post_params.mean = ldlt.solve(....)
```

Reply:
I see that we usually go with LLT. Why LDLT in this case?

Reply:
That's what the stan guys do, I don't think it's going to matter much, ldlt should be more appropriate for large matrices and badly conditioned ones, but that's hardly going to be our case

Reply:
Since it's basically the same, I eventually went with LLT just for consistency's sake